torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2005,6 +2005,30 @@ def sample_inputs_trace(self, device, dtype, requires_grad, **kwargs):
                                      requires_grad=requires_grad))),)
 
 
+def sample_inputs_linalg_trace(self, device, dtype, requires_grad, **kwargs):
+    inputs = (
+        ((S, S), 0),
```

Comment:
We should probably shrink these sample inputs -- they seem very big and that will make the tests take longer than necessary